app/common/services/endpoints/config.js

```diff
@@ -2,24 +2,35 @@ module.exports = [
     '$resource',
     'Util',
     '_',
+    'CacheFactory',
 function (
     $resource,
     Util,
-    _
+    _,
+    CacheFactory
 ) {
+    var cache = new CacheFactory('configCache');
```

Comment:
I don't think you're supposed to `new` CacheFactory.. docs simply have `profileCache = CacheFactory('profileCache');`